pom.xml

```diff
@@ -95,6 +96,36 @@ under the License.
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Thanks for your contribution. I think using [jackson-bom](https://github.com/FasterXML/jackson-bom) here is more clear.

Reply:
Good point. Thanks a lot for your input!